net/mmfd/Makefile

```
@@ -1,10 +1,10 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mmfd
-PKG_SOURCE_DATE:=2019-07-20
+PKG_SOURCE_DATE:=2019-07-21
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL=https://github.com/freifunk-gluon/mmfd.git
-PKG_SOURCE_VERSION:=3c4ce688e40c32e7dfbf945d9f224353fe5d8363
+PKG_SOURCE_VERSION:=810a3448eb444814be536bf08e758831cff4054b
```

```suggestion
PKG_SOURCE_VERSION:=c9e8192919de89087f3996d5746ff06444b6f136
```

You should use https://github.com/freifunk-gluon/mmfd/pull/24